app/api/groups/remove_manager.go

```diff
@@ -66,11 +67,14 @@ func (srv *Service) removeGroupManager(w http.ResponseWriter, r *http.Request) s
 		//    or have can_manage:memberships_and_group permission on the groupID
 		// 2) there should be a row in group_managers for the given groupID-managerID pair
 		if managerID == user.GroupID {
-			found, err = store.GroupManagers().
+			found, err = store.GroupManagers().WithExclusiveWriteLock().
```

Comment:
What's the risk without this lock?

Reply:
Without this lock, the query would not see the rows committed concurrently after the start of the transaction.

Reply:
It was weird to have locking only for one of two cases

Reply:
I think I don't see more reasons for the other lock 😄